### PR TITLE
fix issue 1774

### PR DESCRIPTION
**Pull request by zg009**
changed account manager error checking, changed loadUser function to properly render html, updated unit test

### CODE REVIEW

**Review by bourgeoa — APPROVED** [review]
LGTM you can merge

**Comment by edwardsph** [comment]
I think the message should say the email is sent 'to' the associated account, not from as that doesn't make sense.

**Comment by bourgeoa** [comment]
Thanks. Done

File: lib/requests/password-reset-email-request.js [hunk 1]
```diff
@@ -123,7 +123,10 @@ class PasswordResetEmailRequest extends AuthRequest {
     return this.accountManager.accountExists(username)
       .then(exists => {
         if (!exists) {
-          throw new Error('Account not found for that username')
+          // For security reason avoid leaking error information
```
Comment (TallTed) [inline lib/requests/password-reset-email-request.js:126]: Merged too quickly!
```suggestion
          // For security reasons, avoid leaking error information
```

Comment (bourgeoa) [reply lib/requests/password-reset-email-request.js:126]: Thanks

**Pull request merged**